third_party/requirements.txt

```diff
@@ -11,7 +11,7 @@ exhale==0.3.0
 idna==2.8
 imagesize==1.1.0
 Jinja2==2.10.3
-lxml==4.9.1
+lxml==4.9.2
```

Comment:
why this change?

Reply:
Windows seems happier with this version. That's the only reason, TBH.